packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -279,6 +291,8 @@ function UserMenu({
       university: {
         universityPreferences: `${hosts.university}/edit_profile`,
       },
+      devHub: 'https://developer.mongodb.com',
+      forums: 'https://forums.mongodb.com',
```

Comment:
This link doesn't work. I think you need to be less specific since it's a redirect. `forums.mongodb.com` works, but not `https://forums.mongodb.com`.